scripts/esrupgrades/esr.5.37-7.8.mysql.up.sql

```diff
@@ -0,0 +1,1263 @@
+/* ==> mysql/000054_create_crt_channelmembership_count.up.sql <== */
```

Comment:
For consistency, should we organize these into `mysql/` (and if necessary) `postgres/` folders to mirror the `db/` layout? (And minimize the chance of trying the wrong file? Not that it would run that far :P)